src/hcf-release/jobs/uaa-create-user/templates/run.erb

```diff
@@ -32,9 +43,13 @@ function retry () {
     shift 2
 
     while test ${i} -lt ${max} ; do
+	printf "Trying: %s\n" "$*"
         if "$@" ; then
+	    status ' SUCCESS'
             break
         fi
+	trouble '  FAILED'
+	status "Waiting ${delay} ..."
```

Comment:
The problem with `--silent` is that it also suppresses any error messages from the curl. And then command fails I really want to know that for evaluation of the problem.

And printing something every iteration is IMHO a good thing to show that the role is active, even when not live and ready.